docs/tools/renderware/common/levels.md

```diff
@@ -7,4 +7,9 @@ description: Level editing tools for Renderware Sonic games
 Level editor for Sonic Heroes and Shadow the Hedgehog.
 
 [Download :fontawesome-solid-download:](https://github.com/igorseabra4/HeroesPowerPlant/releases){ .md-button .md-button--primary target="_blank"}
-[GitHub Repository :fontawesome-brands-github:](https://github.com/igorseabra4/HeroesPowerPlant){ .md-button target="_blank"}
\ No newline at end of file
+[GitHub Repository :fontawesome-brands-github:](https://github.com/igorseabra4/HeroesPowerPlant){ .md-button target="_blank"}
+
+## Rail Canyon
+Level viewer/editor for Sonic Heroes.
+
+[GitHub Repository :fontawesome-brands-github:](https://github.com/DeadlyFugu/rail-canyon){ .md-button target="_blank"}
```

Comment:
Any difference between Rail Canyon and HeroesPowerPlant?

If yes, put Rail Canyon on a separate archive for heroes
If not, delete Rail Canyon.